src/sheets/sheets.js

```diff
@@ -1,156 +1,69 @@
 import c from '../config.js'
 import {google} from 'googleapis'
-import {sleep} from './utils.js'
-import logger from '../logger.js'
 
 const scopes = ['https://www.googleapis.com/auth/spreadsheets']
 const key = Buffer.from(c.google.key, 'base64').toString()
 const auth = new google.auth.JWT(c.google.email, null, key, scopes)
 const sheetsApi = google.sheets({version: 'v4', auth})
 
+async function tryCall(call) {
```

Comment:
Is it safe to remove the locks? Or are they simply not worth it?

Reply:
I think it will be fine. They made sense only for updating fields, and these fields are not updated manually very often. And even if they are, manual updates should have higher priority.